packages/ai/tests/ai/common/test_config.py

```
@@ -0,0 +1,110 @@
+sys.modules['rocketlib'] = mock_rocketlib
+sys.modules['depends'] = MagicMock()
+sys.modules['json5'] = MagicMock()
+
+# Now import the module
+from ai.common.config import Config
+
+
+class TestGetNodeConfigDeprecatedProfiles:
+    """Test cases for deprecated profile warning functionality."""
+
+    def _make_service_with_profile(self, profile_name, profile_config, default_profile=None):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_make_service_with_profile`

(ANN202)